src/main/java/org/mafagafogigante/dungeon/commands/CommandSet.java

```diff
@@ -109,12 +108,12 @@ public String toString() {
    * Retrieves a list of the names of the commands closest to the provided token according to their Levenshtein
    * distance.
    */
-  public List<String> getClosestCommands(String token) {
+  List<String> getClosestCommands(final String token) {
     List<String> closestCommands = new ArrayList<>();
     int best = Integer.MAX_VALUE;
     for (Command command : commands) {
       String commandName = command.getDescription().getName();
-      int distance = StringUtils.getLevenshteinDistance(token, commandName);
+      int distance = StringDistanceMetrics.levenshteinDistance(token, commandName);
```

Comment:
This is good because it encapsulates how we are actually computing the LD of the command.